contracts/ResultManager.sol

```diff
@@ -3,6 +3,7 @@ pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts/access/AccessControlEnumerable.sol";
 import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
 
 contract ResultManager is AccessControlEnumerable {
     struct Block {
```

Comment:
remove this struct, this is being used as Value struct already

Reply:
removed https://github.com/razor-network/bridge-proxy/pull/59/commits/00222f0d6f0d9912d66935ac1fd7dde93da0f06b